app/src/main/res/values/strings.xml

```diff
@@ -103,12 +103,120 @@
     <!-- questionnaire -->
     <string name="questionnaire">Questionnaire</string>
     <string name="send_questionnaire">Send Questionnaire</string>
-    <string name="questionnaire_how_old">How old are you?</string>
-    <string name="questionnaire_age_range_10">10~19</string>
-    <string name="questionnaire_age_range_20">20~29</string>
-    <string name="questionnaire_age_range_30">30~39</string>
-    <string name="questionnaire_age_range_40">40~49</string>
-    <string name="questionnaire_age_range_50">50~</string>
+    <string name="questionnaire_other_field_post_value" translatable="false">__other_option__</string>
+    <string name="questionnaire_other_field_label">[need translate]その他</string>
+    <string name="questionnaire_other_field_hint">[need update]optional</string>
+
+    <string name="questionnaire_title_age">Age</string>
+    <string name="questionnaire_description_age">How old are you?</string>
+    <string-array name="questionnaire_labels_age">
+        <item>10~19</item>
+        <item>20~29</item>
+        <item>30~39</item>
+        <item>40~49</item>
+        <item>50~59</item>
+        <item>60~</item>
+        <item>secret</item>
+    </string-array>
+
+    <string name="questionnaire_title_android_user">How long have you been using Android?</string>
+    <string-array name="questionnaire_labels_android_user">
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_labels_android_user appears to be unused [UnusedResources]